docs/synthetics/browser-monitor.md

```diff
@@ -0,0 +1,104 @@
+
+The browser monitor collects the below page load (navigation) metrics for every run. If there are multiple page loads during the execution of the script, the last page load metrics are collected.
+
+| Name  | Label  | Description  | Unit  |
+|---|---|---|---|
+| synthetics.browser.time.frontend | Frontend time | Time taken for the browser to parse and create the page | ms |
+| synthetics.browser.time.backend | Backend time |  Time taken for the network and the server to generate and start sending the HTML | | ms |
+| synthetics.browser.time.pageload | Page load time | Time taken for the page to load, from initiation of the page load (e.g., click on a page link) to load completion in the browser | ms |
+| synthetics.browser.time.dns | DNS time | DNS resolution time for the URL of the page | ms |
+| synthetics.browser.time.connection | Connection time | Time taken to connect to server | ms |
+| synthetics.browser.time.response | Server response time | Time taken for the server to send the response | ms |
+| synthetics.browser.time.download | Page download time | Time taken to download the page contents | ms |
+| synthetics.browser.time.dom.interactive | DOM interactive time | Time taken by the browser to parse the document, including the network time from the user's location to your server | ms |
+| synthetics.browser.time.dom.contentload | DOM content load time | Time taken by the browser to parse the document and execute deferred and parser-inserted scripts including the network time from the user's location to your server | ms |
+| synthetics.browser.time.paint.first | First paint time | The time from navigation to the time when the first paint happens on the screen | ms |
+| synthetics.browser.time.paint.firstcontentful | First contentful paint time |  The time from navigation to the time when the browser renders the first bit of content from the DOM | ms |
+
+#### Resource Metrics
+
+Browser monitor collects the below metrics for every resource loaded during the execution of the script:
+
+| Name  | Label  | Description  | Unit  |
+|---|---|---|---|
+| synthetics.browser.resource.transfer.size | Transfer size | Network size of the resource content | bytes |
+| synthetics.browser.resource.time | Resource time | Total time taken to fetch the resource | ms |
+
+### Screenshots
+
+The browser monitor script allows the collection of the screenshot of the page at any point during the execution. This can be collected using [page.screenshot()](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagescreenshotoptions) Puppeteer API. JPEG and PNG image types are supported. Currently, the number of screenshots per run is limited to 1.
+
+### Waterfall chart
+
+For every run, the browser monitor collects all the resources fetched during the run. These resources are shown in a graphical waterfall chart in the individual run details page. The metric shown for each resource are:
+
+* **Total Time** - Total time to fetch the resource in ms
```

Comment:
This is fine, but just curious why we didn't display these in a table, like we did with metrics above?